src/js/Controllers/DisplayCapabilities.js

```diff
@@ -32,12 +32,13 @@ let screenParams = {
 	}
 }
 
-function textField(name, width, rows) {
+function textField(name, width, rows, characterSet) {
 	width = width ? width : 500
 	rows = rows ? rows : 1
+	characterSet = characterSet ? characterSet : "UTF_8"
```

Comment:
```suggestion
function textField(name, width=500, rows=1, characterSet="UTF_8") {
```

what do you think about default params?